mirdata/datasets/da_tacos.py

```diff
@@ -0,0 +1,640 @@
+
+    @core.cached_property
+    def metadata(self) -> dict:
+        return self._track_metadata
+
+    @core.cached_property
+    def cens(self) -> np.array:
+        return load_cens(self.cens_path)
+
+    @core.cached_property
+    def crema(self) -> np.array:
+        return load_crema(self.crema_path)
+
+    @core.cached_property
+    def hpcp(self) -> np.array:
+        return load_hpcp(self.hpcp_path)
+
+    @core.cached_property
+    def key(self) -> dict:
+        return load_key(self.key_path)
+
+    @core.cached_property
+    def madmom(self) -> np.array:
+        return load_madmom(self.madmom_path)
+
+    @core.cached_property
+    def mfcc(self) -> np.array:
+        return load_mfcc(self.mfcc_path)
+
+    @core.cached_property
+    def tags(self) -> list:
+        return load_tags(self.tags_path)
+
+    def to_jams(self) -> JAMS:
+        """Get the track's data in jams format
+
+        Returns:
+            jams.JAMS: the track's data in jams format
+
+        """
+        return jams_utils.jams_converter(
+            metadata={
+                "duration": 0.0,
+                "work_id": self.work_id,
+                "performance_id": self.performance_id,
+                "subset": self.subset,
+                "label": self.label,
+                "cens": self.cens,
+                "crema": self.crema,
+                "hpcp": self.hpcp,
+                "key": self.key,
+                "madmom": self.madmom,
+                "mfcc": self.mfcc,
+                "tags": self.tags,
+            }
+        )
+
+
+def load_cens(path):
+    """Load da_tacos cens features from a file
+
+    Args:
+        path(str or file-like): File-like object or path to features file
+
+    Returns:
+        np.array: cens features
+
+    """
+    return dd.io.load(path)["chroma_cens"]
+
+
+def load_crema(path):
+    """Load da_tacos crema features from a file
+
+    Args:
+        path(str or file-like): File-like object or path to features file
+
+    Returns:
+        np.array: crema features
+
+    """
+    return dd.io.load(path)["crema"]
+
+
+def load_hpcp(path):
+    """Load da_tacos hpcp features from a file
+
+    Args:
+        path(str or file-like): File-like object or path to features file
+
+    Returns:
+        np.array: hpcp features
+
+    """
+    return dd.io.load(path)["hpcp"]
+
+
+def load_key(path):
+    """Load da_tacos key features from a file.
+
+    Args:
+        path(str or file-like): File-like object or path to features file
+
+    Returns:
+        dict: key
+
+    Examples:
+        {'key': 'C', 'scale': 'major', 'strength': 0.8449875116348267}
+
+    """
+    return dd.io.load(path)["key_extractor"]
+
+
+def load_madmom(path):
+    """Load da_tacos madmom features from a file
+
+    Args:
+        path(str or file-like): File-like object or path to features file
+
+    Returns:
+        dict: madmom features
+
+    Examples:
+        {
+           "novfn":"array("[
```

Comment:
Can we change the nested array views in these examples? Lots of unnecessary lines I think.

Reply:
Hi @furkanyesiler! Ok perfect!

Reply:
done 764146f